model/FullText.inc.php

```diff
@@ -52,58 +53,33 @@ public static function indexItem(Zotero_Item $item, $data) {
 			. implode(', ', array_fill(0, sizeOf($params), '?')) . ")";
 		Zotero_DB::query($sql, $params, Zotero_Shards::getByLibraryID($libraryID));
 		
-		// Add to Elasticsearch
-		self::indexItemInElasticsearch($libraryID, $key, $version, $timestamp, $data);
-		
-		Zotero_DB::commit();
-	}
-	
-	private static function indexItemInElasticsearch($libraryID, $key, $version, $timestamp, $data) {
-		$type = self::getWriteType();
-		
-		$id = $libraryID . "/" . $key;
-		$doc = [
-			'id' => $id,
+		// Add to S3
+		$json = [
 			'libraryID' => $libraryID,
-			'content' => (string) $data->content,
-			// We don't seem to be able to search on _version, so we duplicate it here
+			'key' => $key,
 			'version' => $version,
-			// Add "T" between date and time for Elasticsearch
+			'content' => (string)$data->content,
```

Comment:
Yeah, I thought about this, but I automatically reformat updated code parts before commit, and couldn't find how to add space here. Ok, I'll try harder 😄